lib/lower/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-error")
```

Comment:
Can the warnings be fixed or suppressed?

Reply:
I cannot fixed those easily inside LLVM headers. Alos, I could certainly not fix them in released LLVM versions. However, I found a way to tell compilers to ignore warnings from headers depending on which directory they live in: adding `SYSTEM` to the related `include_directories`. It worked in my tests with clang 8 and gcc 8.3 on linux.

The drone integration are still failing with clang, but this now has to do with ZLIB linking issues and not unused parameter warnings anymore.

Reply:
With that change you can remove the -Wno-error flag here.

Reply:
Good point, thanks. Done.